app/main.py

```diff
@@ -2,0 +1,13 @@
+def copy_file(command: str) -> None:
+    cp, old_file, new_file = command.split()
+
+    if cp != "cp":
+        raise ValueError("The command need to start with 'cp'")
+
+    if command.split() != 3:
+        raise ValueError("Command format needs to be <cp file_1 file_2>")
```

Comment:
There is an issue with the condition in this line. You are trying to split the command again and compare the result with 3. However, `split()` method returns a list, and you are comparing it with an integer. This will always return False. You probably wanted to check the length of the list, so you should use `len(command.split())` instead.